### ISSUE TITLE
Is NODE_ENV replaced by string 'none' during nest start execution!

### ISSUE THREAD

**Issue by iamsteeve** [nestjs/nest-cli#420]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ x ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

It is exactly a problem that affects projects that use NODE_ENV. The problem is that when changing the project to monorepo mode and when executing npm run start: dev (nest start --watch), NODE_ENV is filled in with a string that says "none" during execution instead of being undefined and cannot be replaced

## Expected behavior

NODE_ENV = undefined
It is an important environment variable since it indicates which environment to run.

## Minimal reproduction of the problem with instructions

1.- nest new project
2.- cd project
3.- nest g lib integrations
4.- npm run start:dev (nest start --watch)
NODE_ENV === 'none'
## What is the motivation / use case for changing the behavior?

Without NODE_ENV I do not execute the project :c and I am integrating monorepo.

## Environment

```
[System Information]
OS Version     : macOS Catalina
NodeJS Version : v10.16.3
NPM Version    : 6.12.0
[Nest Information]
platform-express version : 6.7.2
common version           : 6.7.2
core version             : 6.7.2
```

**Comment by iamsteeve** [comment]
https://github.com/webpack/webpack/issues/7470
nodeEnv?
I think this is the problem, xD although I have no idea if it is the cli or core of nest xD I will clone the projects in any case to verify xD (excuse my English I am Mexican :p)

**Comment by kamilmysliwiec** [comment]
Fixed in 6.11.0
